sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/transition/ViewportTransitionOptions.kt

```diff
@@ -0,0 +1,78 @@
+package com.mapbox.maps.plugin.viewport.experimental.transition
+
+import com.mapbox.maps.plugin.viewport.DEFAULT_FRAME_TRANSITION_MAX_DURATION_MS
+
+/**
+ * Options that impact the transition animation.
+ *
+ * The styling of the animation is determined by the [ViewportStateTransition]
+ * and [ViewportTransition] implementations, but options here provide higher-level
+ * constraints that those implementations need to obey.
+ *
+ * @param maxDurationMs maximum duration of the generated transitions set,
+ * including delays between animators and their respective durations.
+ */
+class ViewportTransitionOptions private constructor(
+  val maxDurationMs: Long
```

Comment:
It'd be good to consider a `minDurationMs` that would extend transitions where needed.

Reply:
The ViewportTransitionOptions is now moved to `FollowingViewportStateOptions` , `OverviewViewportStateOptions` and `DefaultViewportTransitionOptions` separately. Would be good to consider adding later incrementally without breaking change.